.github/workflows/lint.yml

```diff
@@ -0,0 +1,21 @@
+name: Lint rules
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: ricardochaves/python-lint@v1.1.0
+        with:
+          python-root-list: "python_fluentd_testing tests"
+          use-black: true
+          use-mypy: true
+          use-isort: true
```

Comment:
true is default

Reply:
Good to know it! I believe I'll leave them that way to be explicit about what I want. I would have to see the project to understand it or see previous builds.